### DISCUSSION TITLE
YouTube Shorts 已有超過 15 億月活躍用戶

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#453] [General]
YouTube Shorts 已有超過 15 億月活躍用戶
現在常常Youtube看一看就滑過去Shorts，一看就是好幾十分鐘
https://tw.news.yahoo.com/youtube-shorts-%E5%B7%B2%E6%9C%89%E8%B6%85%E9%81%8E-15-%E5%84%84%E6%9C%88%E6%B4%BB%E8%BA%8D%E7%94%A8%E6%88%B6-023055507.html

**Comment by brian431** [comment]
> YouTube Shorts 已有超過 15 億月活躍用戶
> 
> 現在常常Youtube看一看就滑過去Shorts，一看就是好幾十分鐘
> 
> https://tw.news.yahoo.com/youtube-shorts-%E5%B7%B2%E6%9C%89%E8%B6%85%E9%81%8E-15-%E5%84%84%E6%9C%88%E6%B4%BB%E8%BA%8D%E7%94%A8%E6%88%B6-023055507.html

抖音及YouTube Shorts的成功顯示出了人們普遍喜歡較短的影片，或許跟注意力的集中時間有所關聯。
